mlflow/genai/labeling/__init__.py

```diff
@@ -63,21 +63,29 @@ def get_labeling_sessions() -> list[LabelingSession]:
     return get_review_app().get_labeling_sessions()
 
 
-def get_labeling_session(name: str) -> LabelingSession:
+def get_labeling_session(
+    name: Optional[str] = None, run_id: Optional[str] = None
+) -> LabelingSession:
```

Comment:
I think we should only accept run ID, rather than 2 optional parameters, since name is not necessarily unique.

cc @epec254